app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -53,6 +53,11 @@ def undestroy
         render :json => item
       end
 
+      def next_name
+        svc = Catalog::NextName.new(params.require(:portfolio_item_id), params[:portfolio_id])
```

Comment:
Do you think there's any merit in changing the parameter `portfolio_id` to `destination_portfolio_id` to kinda match the openapi.json? The openapi.json `name` would also have to change, but at first when I was looking through this PR I saw the extra parameter and I didn't realize it was here.

Reply:
Sure, I like that idea, updated.